README.md

```diff
@@ -1,24 +1,75 @@
-# README
+# Hello Rails Back End
```

Comment:
- [ ] I was unable to properly fetch greetings from the API due to CORS issue.

  ![err-cor](https://user-images.githubusercontent.com/36542986/184106168-5fc26afd-9785-4da7-9cb1-7d5fb482659c.jpg)

  You can fix this by installing the `rack-cors` gem and creating the `config/initializers/cors.rb` file and pasting the code below in it.

  ```
  Rails.application.config.middleware.insert_before 0, Rack::Cors do
    allow do
      origins '*'
      resource '*', headers: :any, methods: [:get, :post, :patch, :put]
    end
  end
  ```
  You can visit [this link for more](https://github.com/cyu/rack-cors)

Reply:
Thank you for the feedback. I will implement it!